integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiCommonMount.java

```diff
@@ -80,19 +100,23 @@
    *        JUnit engine parameter resolution mechanism
    */
   @BeforeAll
-  public static void initAll(@Namespaces(2) List<String> namespaces) {
+  public static void initAll(@Namespaces(3) List<String> namespaces) {
     logger = getLogger();
     // get a new unique opNamespace
     logger.info("Creating unique namespace for Operator");
     assertNotNull(namespaces.get(0), "Namespace list is null");
     opNamespace = namespaces.get(0);
 
-    logger.info("Creating unique namespace for Domain");
+    logger.info("Creating unique namespace for Domain1");
     assertNotNull(namespaces.get(1), "Namespace list is null");
-    domainNamespace = namespaces.get(1);
+    domain1Namespace = namespaces.get(1);
+
+    logger.info("Creating unique namespace for Domain2");
+    assertNotNull(namespaces.get(2), "Namespace list is null");
+    domain2Namespace = namespaces.get(2);
```

Comment:
Why we need 2 domains in this usecases ? If needed please explain on the javadoc section of the test method.

Reply:
just to make the tests independent from each other.

Reply:
I would rather prefer merge the usecase to avoid multiple domain boot if possible, you can make the Data Source uecase as  additional common mount in addition to the existing test method.

Reply:
changed test to have 1 domain